src/com/carolinarollergirls/scoreboard/core/impl/TeamImpl.java

```diff
@@ -61,6 +64,17 @@ protected Object computeValue(PermanentProperty prop, Object value, Object last,
         if (prop == Value.OFFICIAL_REVIEWS && (Integer)value > scoreBoard.getRulesets().getInt(Rule.NUMBER_REVIEWS)) {
             return scoreBoard.getRulesets().getInt(Rule.NUMBER_REVIEWS);
         }
+        if (prop == Value.TRIP_SCORE && flag != Flag.COPY && scoreBoard.isInJam() && (Integer)value > 0) {
+            tripScoreTimerTask.cancel();
```

Comment:
Should we also clear it when a trip is added by hand?

Reply:
Good idea. I also added it when a trip is removed by hand.